src/libs/actions/IOU.ts

```diff
@@ -8250,7 +8319,7 @@ function getNextApproverAccountID(report: OnyxEntry<OnyxTypes.Report>, isUnappro
     return getAccountIDsByLogins([nextApproverEmail]).at(0);
 }
 
-function approveMoneyRequest(expenseReport: OnyxEntry<OnyxTypes.Report>, full?: boolean) {
+function approveMoneyRequest(expenseReport: OnyxEntry<OnyxTypes.Report>, full?: boolean, hash?: number) {
```

Comment:
I think it's a mistake to add optional `hash` to all these actions that are report centric.

These functions should not know about the existence of Search. Up until now everything worked nicely with these functions being separate, and now because of adding an animation when something is removed we are combining them together.
I would consider a different approach.